init.lua

```diff
@@ -363,4 +366,10 @@ function throwing.register_bow(name, def)
 		return itemstack
 	end
 	minetest.register_tool(name, def)
+	if use_toolranks then
```

Comment:
It is probably undesirable to enforce the use of `toolranks` for all items. An optional bow definition boolean parameter `use_toolranks` defaulting to `true` would be better, so that the use of toolranks can be disabled when registering a bow!